pkg/kubelet/metrics/metrics.go

```diff
@@ -165,17 +165,16 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
-	// PLEGDiscardEvents is a Histogram that tracks the duration (in seconds) it takes for discarding events in the Kubelet's
-	// Pod Lifecycle Event Generator (PLEG).
-	PLEGDiscardEvents = metrics.NewCounterVec(
+	// PLEGDiscardEvents is a Counter that tracks the number of discarding events in the Kubelet's Pod Lifecycle Event Generator (PLEG).
+	PLEGDiscardEvents = metrics.NewCounter(
```

Comment:
Do we need to go through some form of deprecation process to change the type of this metric? I.e. would this change not cause breakages for anyone who currently consumes `PLEGDiscardEvents`?

Does the `StabilityLevel: metrics.ALPHA` designation have any impact on my question above?

Reply:
ALPHA metrics have no stability guarantees，and it just use in PLEG relist() function, so no impact for other one

Reply:
:+1: